docs/how-to/ChIP_seq_genomic_ranges_tutorial.ipynb

```diff
@@ -0,0 +1,2733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    genes["seq_name"] = genes["seq_name"].map(hg38.alias_dict)
</pre><p>With <code>pd.Series.map</code> any value in the series that isn't in the dict will be mapped to a null value.</p><p> </p><p>Should this be:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">genes["seq_name"] = genes["seq_name"].replace(hg38.alias_dict)
</pre><p>instead? Or should non-standard contigs even be filtered out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/genomic-features/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>They would probably anyways be filtered out in the next step, if they do not overlap?</p>

Reply:
<p>The unmapped chromosome calls are given NA values that are then filtered using the dropna() function.</p>